### PR TITLE
Sum of Totient Function code rfc

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

I just got rid of macros functions.

AC: https://judge.yosupo.jp/submission/244500

### CODE REVIEW

File: content/6_Advanced/Multiplicative.mdx [hunk 1]
```diff
@@ -218,19 +218,240 @@ identity function $I$. Sometimes, a more careful observation is needed.
 <CPPSection>
 
 ```cpp
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+
+// BeginCodeSnip{ModInt}
```
Comment (SansPapyrus683) [inline content/6_Advanced/Multiplicative.mdx:225]: where's this from

Comment (freakin23) [reply content/6_Advanced/Multiplicative.mdx:225]: It's the one I use, which belongs to tourist

Comment (SansPapyrus683) [reply content/6_Advanced/Multiplicative.mdx:225]: surely there's a shorter one that suffices for this code 💀

Comment (SansPapyrus683) [reply content/6_Advanced/Multiplicative.mdx:225]: or wait does the old code just implicitly use this

Comment (freakin23) [reply content/6_Advanced/Multiplicative.mdx:225]: The previous author of the code was using such a template but it was not mentioned there. So I just added what I use often.

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**